rdm/main.py

```diff
@@ -51,6 +52,8 @@ def cli(raw_arguments):
             exit_code = 1
     elif args.command == 'translate':
         translate_test_results(args.format, args.input, args.output)
+    elif args.command == 'gap':
+        audit_for_gaps(args)
```

Comment:
It would be useful if the exist code were set to 0 if the audit passes, and 1 if it fails.

Reply:
Also, perhaps we should add a pytest that audits our base templates to see if they pass a 62304 and 14971 gap analysis?

Reply:
See https://github.com/innolitics/rdm/blob/master/tests/fresh_release_test.py#L31

You could probably add the audit checks to the existing integration test. IMO with integration tests, it is better to have one big test vs many independent tests (like you usually want with small fast unit tests).

Reply:
I added exit codes with 0 for success.

I think adding a pytest would be better to do as part of a separate issue+PR to pass those tests as those are large tasks all by themselves.

Reply:
> I think adding a pytest would be better to do as part of a separate issue+PR to pass those tests as those are large tasks all by themselves.

sounds good!